kyo-core/shared/src/test/scala/kyo/ChannelTest.scala

```diff
@@ -780,13 +780,15 @@ class ChannelTest extends Test:
         }
 
         "should stream concurrently with ingest via putBatch, yielding consistent chunk sizes" in run {
+            pending
```

Comment:
@johnhungerford This test is flaky. I was able to reproduce the failure locally with and without this PR but it's rare. It seems the scenario you were worried about. Since `channel.stream` takes a `maxChunkSize`, is it reasonable that chunks of smaller size may arrive? It's difficult to debug but it seems `putBatch` is still adding items to the queue while `drainUpTo` is executing and it finalizes earlier before `putBatch` is able to place all items in the queue. It seems a reasonable behavior to me, maybe we can remove the test

Reply:
This should probably just be like the other concurrent streaming test, which only asserts that every batch size is `<=` to `maxChunkSize` and that the flattened result contains all expected values ordered correctly. As you say it's reasonable that chunk sizes are not guaranteed, but we should confirm everything else works when concurrently streaming in and out.